skins/monokai.yml

```diff
@@ -37,38 +42,45 @@ k9s:
     buttonFocusBgColor: *darkBlue
     labelFgColor: *orange
     fieldFgColor: *blue
+
   frame:
     # Borders styles.
     border:
       fgColor: *blue
       focusColor: *darkBlue
+      
     menu:
       fgColor: *lightBlue
       keyColor: *green
       # Used for favorite namespaces
       numKeyColor: *green
+
     # CrumbView attributes for history navigation.
     crumbs:
       fgColor: *foreground
       bgColor: *backgroundOpaque
       activeColor: *blue
+
     # Resource status and update styles
     status:
       newColor: *blue
       modifyColor: *purple
       addColor: *green
       pendingColor: *orange
       errorColor: *magenta
-      highlightColor: *blue
+      highlightcolor: *blue
```

Comment:
Why did you change the key here? I think it's wrong: https://github.com/derailed/k9s/blob/master/internal/config/styles.go#L110

Reply:
Okay, I fixed that, but I also went through each section in that styles.go and made sure there wasn't anything else missing. I also tweaked the colors because it wasn't really "monokai" enough for my liking.